pkgs/os-specific/linux/mouse_m908/default.nix

```diff
@@ -0,0 +1,45 @@
+{ stdenv
+, lib
+, libusb
+, fetchFromGitHub
+, pkg-config
+}: stdenv.mkDerivation {
+  name = "mouse_m908";
+  version = "3.4";
+
+  nativeBuildInputs = [ pkg-config ];
+
+  buildInputs = [ libusb.dev ];
+
+  # Uses proper nix directories rather than the ones specified in the makefile
+  installPhase = ''
+    mkdir -p $out/bin \
+      $out/share/doc \
+      $out/share/man/man1 \
+      $out/share/doc/mouse_m908 \
+      $out/lib/udev/rules.d
+
+    cp mouse_m908 $out/bin/mouse_m908 && \
+    cp mouse_m908.rules $out/lib/udev/rules.d && \
+    cp examples/* $out/share/doc/mouse_m908 && \
+    cp README.md $out/share/doc/mouse_m908 && \
+    cp keymap.md $out/share/doc/mouse_m908 && \
+    cp mouse_m908.1 $out/share/man/man1
+  '';
+
+  src = fetchFromGitHub {
+    owner = "dokutan";
+    repo = "mouse_m908";
+    rev = "6e48ff76cfe2e199215e4c9a916fbc34619ab227";
```

Comment:
Please reference version (`stdenv.mkDerivation (finalAttrs: {...})`) and `rev = "v${finalAttrs.version}";`